### DISCUSSION TITLE
No response after installing

### DISCUSSION THREAD

**Discussion by JGiubardo** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#782] [Q&A]
I have a Hawaii S200106 and a V2 board, I was able to configure it on my PC but have no lights on the display or the board after installing it in the egg.
![image](https://github.com/user-attachments/assets/1ba3b042-691b-4fcc-b62d-203928da3072)
![image](https://github.com/user-attachments/assets/05db6b09-b3dc-4343-a987-2ea6946931c3)
![image](https://github.com/user-attachments/assets/38aac502-f402-4118-82af-885ce3957026)

**Comment by visualapproach** [comment]
Firmware?

**Reply by JGiubardo** [reply]
Firmware: 2024-07-23-1743
Model: MALDIVES2021

**Reply by visualapproach** [reply]
Ok, the model isn't listed as compatible. Doesn't mean it isn't but there is a possibility. You could run the debug-on, hwtest and inputs tests and see if there is any clues

**Reply by JGiubardo** [reply]
Results of hwtest (I ran it with USB power since I'm using the V2 board):
```
Start test. Seq begins with HIGH, then alters.

Sending on D1, receiving on D4
---------------------------------------------------------------------------------------------------- // 0 errors out of 100
Sending on D2, receiving on D5
-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0-0 // 50 errors out of 100
Sending on D3, receiving on D6
---------------------------------------------------------------------------------------------------- // 0 errors out of 100
Sending on D4, receiving on D1
---------------------------------------------------------------------------------------------------- // 0 errors out of 100
Sending on D5, receiving on D2
1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1-1- // 50 errors out of 100
Sending on D6, receiving on D3
---------------------------------------------------------------------------------------------------- // 0 errors out of 100
End of test!
"1" or "0" indicates ERROR, depending on test state. "-" is good.
Switching cio pins 5s HIGH -> 5s LOW -> input
then DSP pins 5s HIGH -> 5s LOW -> input, repeating
Disconnect cables then reset chip when done!
```

How do I access the inputs test and debug mode?

**Reply by visualapproach** [reply]
First you need to fix the connection between D2/D5

**Reply by JGiubardo** [reply]
I can't seem to find any issues with checking continuity with my multimeter.  Is there something more specific I should be looking for?

**Comment by metaljay** [comment]
Aren’t you missing the 10k resistor too?

**Reply by JGiubardo** [reply]
I'll add one

**Reply by visualapproach** [reply]
It *should* work anyways. Maybe LLC or NodeMCU is broken?
